### ISSUE TITLE
Pelvic ring fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11073]
**Preferred term label:**
Pelvic ring fracture

**Synonyms**
open book pelvis; windswept pelvis

**Definition (free text, please give PubMed ID)**
X-ray of the pelvis (AP, inlet, and outlet views) showing fracture of the pelvic ring, including iliac wing (ilium), superior or inferior pubic ramus, or ischium, and/or displacement of the pubic symphysis or sacroiliac joints 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0041161

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pelvic Fracture
Scapula fracture (I am checking into why this makes sense but it is listed at https://www.orthobullets.com/trauma/1013/scapula-fractures "pelvic ring/acetabular fractures (15%); scapula fracture is important predictor"

**Your nano-attribution (ORCID)**
